R/execution.r

```diff
@@ -78,11 +78,17 @@ Executor <- setRefClass(
 is.silent = function() {
     current_request$content$silent
 },
+is.store_history = function() {
+    current_request$content$store_history
+},
+request_metadata.get = function() {
+    current_request$metadata 
+},
 
 send_error_msg = function(msg) {
-    if (!is_silent()) return()
```

Comment:
Yeah but it sounds like people might want to move to an underscore naming scheme anyway :D.